tests/Controller/EnMarche/CitizenActionManagerControllerTest.php

```diff
@@ -125,6 +126,9 @@ public function testCreateCitizenActionSuccessful()
         $this->assertInstanceOf(CitizenAction::class, $citizenAction = $this->getCitizenActionRepository()->findOneBy(['slug' => (new \DateTime())->format('Y-m-d').'-mon-action-citoyenne']));
         $this->assertSame('Mon Action Citoyenne', $citizenAction->getName());
         $this->assertCountMails(0, EventRegistrationConfirmationMessage::class, 'jacques.picard@en-marche.fr');
+        $this->assertCountMails(1, CitizenActionNotificationMessage::class, 'jacques.picard@en-marche.fr');
+        $this->assertCountMails(1, CitizenActionNotificationMessage::class, 'gisele-berthoux@caramail.com');
+        $this->assertCountMails(1, CitizenActionNotificationMessage::class, 'luciole1989@spambox.fr');
```

Comment:
Let's add a random (hard-coded not really random) person(s) that is not supposed to receive this email ? Not sure it's really useful :D

Reply:
" it's free \o/ "